assimilation_code/modules/utilities/netcdf_utilities_mod.f90

```diff
@@ -64,7 +62,8 @@ module netcdf_utilities_mod
           nc_close_file,                  &
           nc_begin_define_mode,           &
           nc_end_define_mode,             &
-          nc_synchronize_file
+          nc_synchronize_file,            &
+          NF90_MAX_NAME, NF90_MAX_VAR_DIMS
```

Comment:
Passing these variables through allows us to declare variables of the right length without having a `use netcdf` or `use typesizes` statement.

Reply:
this is great - i didn't think to do that and it does let you set the lengths using the actual netcdf variables.  one question: would it be less confusing to define our own parameters which are initialized to these lengths and then export those?  otherwise it could be unclear how they get values if the user isn't using the actual netcdf module?

Reply:
I'm still a bit torn. If this thing is identically described in the netCDF Fortran manuals, I'd like to keep it the same. If someone web-searched it, it would be clear. I guess I'd rather not create something exactly the same and just call it something different.

Reply:
`use netcdf_utilities_mod,  only :` I think covers where NF90_MAX_NAME comes from.

Reply:
Helen, are you agreeing that we should keep the name the same? Or are you raising a different issue?

Reply:
Agreeing that we keep the same name.